### DISCUSSION TITLE
Storybook 8.0.0-beta.0 missing global __STORYBOOK_CLIENT_API__

### DISCUSSION THREAD

**Discussion by khats** [storybookjs/storybook#25892] [Help]
### Summary

We were using following method to access in preview frame to currentStoryName and currentStoryArgs in version 7.6:
`__STORYBOOK_CLIENT_API__.storyStore.args.argsByStoryId`
Looks like this api dropped in version 8

Is there a way in version 8 to access store instance?

**Comment by khats** [comment]
As i see it was removed in scope of removing other deprecated apis :( 
https://github.com/storybookjs/storybook/pull/24528

**Comment by valentinpalkovic** [comment]
@ndelangen might have an idea.

**Comment by ndelangen** [comment]
You might be able to use `__STORYBOOK_STORY_STORE__` for SB8, but that too will eventually disappear too.

If you tell us about your use-case, we might be able to come up with a supported way to do what you want to do.

**Comment by khats** [comment]
@ndelangen use case:
-  we have control with list of layout state(enum)
-  we have auto test step that muting CSS animations, but if control selected value = XXX not muting

So for all scenarios its muting, but for 1  not and this check relies on `__STORYBOOK_CLIENT_API__.storyStore.args.argsByStoryId`. 

But i split this scenarios to avoid condition check under autotest.
9 scenarios with direct muting
1 just not calling mute

So for me issues solved
